src/amo/components/Errors/AuthExpired/styles.scss

```diff
@@ -0,0 +1,7 @@
+@import '~amo/css/styles';
+
+.AuthExpired .Button {
```

Comment:
It would probably be better to add a class to the button itself, as opposed to the whole component.